### PR TITLE
add drycc repo

### PR DESCRIPTION
**Pull request by duanhongyi**
Maintainer of drycc Workflow charts (https://github.com/drycc) here. We would like to be a part of the Helm Hub. Our main charts repo is charts.drycc.cc/stable .

### CODE REVIEW

File: repos.yaml [hunk 1]
```diff
@@ -1620,3 +1620,8 @@ repositories:
     maintainers:
       - email: scott@r6by.com
         name: scottrigby
+  - name: drycc
+    url: https://charts.drycc.cc/stable
```
Comment (mattfarina) [inline repos.yaml:1624]: Can you add this same information to the file at _config/helm-hub-values.yaml_

**Review by mattfarina — CHANGES_REQUESTED** [review]
Thank you for the pull request. There is one change that needs to be made in order for it to be listed...

File: repos.yaml (hunk 1 above)
Comment (duanhongyi) [reply repos.yaml:1624]: ok

**Review by mattfarina — APPROVED** [review]
lgtm

**Pull request merged**